components/specification/released-schema/2016-DEV0/ome.xsd

```diff
@@ -3882,7 +3885,7 @@ PlaneInfo (as Plane)
       <xsd:complexContent>
         <xsd:extension base="Shape">
           <xsd:sequence>
-            <xsd:element ref="BinData" minOccurs="1" maxOccurs="unbounded"/>
+            <xsd:element ref="BinData" minOccurs="1" maxOccurs="1"/>
```

Comment:
@rleigh-dundee: what would be the impact of this additional constraint on our samples using multiple BinData elements? Given the impact of these changes when it comes to propagating across the stack, I am inclined to focus our efforts on `MaskBinData` as initially proposed and have a separate discussion about `PixelsBinData`.